<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="os_groupBox">
       <property name="title">
        <string>OS(Exclusive)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="windows_checkBox">
          <property name="text">
           <string>Windows</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="linux_checkBox">
          <property name="text">
           <string>Linux</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mac_checkBox">
          <property name="text">
           <string>Mac</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="drink_groupBox">
       <property name="title">
        <string>Drink (Non Exclusive)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="milk_checkBox">
          <property name="text">
           <string>Milk</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="tea_checkBox">
          <property name="text">
           <string>Tea</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="water_checkBox">
          <property name="text">
           <string>Water</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="color_groupBox">
     <property name="title">
      <string>Which Color</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="red_radioButton">
        <property name="text">
         <string>Red</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="green_radioButton">
        <property name="text">
         <string>Green</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="blue_radioButton">
        <property name="text">
         <string>Blue</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="get_data_button">
     <property name="text">
      <string>Get Data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="set_data_button">
     <property name="text">
      <string>Set Data</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
